package-lock.json

```diff
@@ -0,0 +1,4544 @@
+{
+  "name": "SpongeDocs",
+  "version": "0.0.1",
```

Comment:
Can we increment this version to something more meaningful? Such as the API version.

Reply:
this file is auto generated and is not modified manually, it basically makes a snapshot of all your installed packages

and the version here is pulled from package.json https://github.com/SpongePowered/SpongeDocs/blob/stable/package.json#L3

Reply:
That is what I meant (update version in `package.json`).